src/lib/list/selection-list.spec.ts

```diff
@@ -628,6 +628,21 @@ describe('MatSelectionList without forms', () => {
       expect(selectList.selected.length).toBe(0);
     });
 
+    it('should not allow select all items using ctrl + a', () => {
+        const event = createKeyboardEvent('keydown', A, selectionList.nativeElement);
+        Object.defineProperty(event, 'ctrlKey', {get: () => true});
+
+        let selectList =
```

Comment:
This should be a `const` since it isn't being re-assigned.

Reply:
Fixed, thanks! :)